test/parser/JavaMetrics.test.ts

```diff
@@ -50,6 +50,13 @@ describe("Java metrics tests.", () => {
                 0
             );
         });
```

Comment:
Nit: add line break here. Also on the other cases where there is no line break between it (...) calls.